src/Microsoft.OpenApi/Models/AuthenticationScheme.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. 
+
+using Microsoft.OpenApi.Attributes;
+
+namespace Microsoft.OpenApi.Models
+{
+    /// <summary>
+    /// The type of HTTP authentication schemes
+    /// </summary>
+    public enum AuthenticationScheme
```

Comment:
This is a thing of beauty.  However, we can't do it this way.  The scheme property is defined as a string in the spec https://spec.openapis.org/oas/v3.0.0#fixed-fields-22 and the [IANA HTTP Auth Scheme](https://www.iana.org/assignments/http-authschemes/http-authschemes.xhtml) registry defines the allow values.  That registry can have new values added at any time.
Also, changing a public member from string to enum is a breaking change, so we couldn't do it in 1.3.

Reply:
I think it should be sufficient to add Bearer and Basic as constants in this file https://github.com/microsoft/OpenAPI.NET/blob/649f0b1780c575c4780f4091eedd7a555e8ccc4a/src/Microsoft.OpenApi/Models/OpenApiConstants.cs

Reply:
Thank you for pointing this out. Resolving..